docs/releases/1.9.txt

```diff
@@ -201,6 +201,10 @@ Management Commands
 * The :djadmin:`createcachetable` command now has a ``--dry-run`` flag to
   print out the SQL rather than execute it.
 
+* When using the PostgreSQL backend, the :djadmin:`dbshell` command no longer
```

Comment:
... command can connect to the database using the password from your settings file (instead of requiring it to be manually entered).